test/Dialect/Handshake/tuple-ops.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: circt-opt -split-input-file --mlir-print-op-generic %s | circt-opt | FileCheck %s
+
+// CHECK-LABEL: handshake.func @unpack_pack(
+// CHECK-SAME:                            %[[VAL_0:.*]]: tuple<i64, i32, i64>,
+// CHECK-SAME:                            %[[VAL_1:.*]]: none, ...) -> (tuple<i32, i64>, none) attributes {argNames = ["in0", "inCtrl"], resNames = ["out0", "outCtrl"]} {
+// CHECK:  %[[VAL_2:.*]]:3 = unpack %[[VAL_0]] : tuple<i64, i32, i64>
+// CHECK:  %[[VAL_3:.*]] = arith.addi %[[VAL_2]]#0, %[[VAL_2]]#2 : i64
+// CHECK:  %[[VAL_4:.*]] = pack %[[VAL_2]]#1, %[[VAL_3]] : tuple<i32, i64>
+// CHECK:  return %[[VAL_4]], %[[VAL_1]] : tuple<i32, i64>, none
+// CHECK:}
+
+handshake.func @unpack_pack(%in: tuple<i64, i32, i64>, %arg1: none, ...) -> (tuple<i32, i64>, none) attributes {argNames = ["in0", "inCtrl"], resNames = ["out0", "outCtrl"]} {
+  %a, %b, %c = handshake.unpack %in : tuple<i64,i32,i64>
```

Comment:
Could also add one more input/set of pack/unpacks to ensure that the attribute dict is printed/parsed correctly.